pkg/option/config.go

```diff
@@ -3279,6 +3283,19 @@ func (c *DaemonConfig) Populate(vp *viper.Viper) {
 			ToFQDNsMaxDeferredConnectionDeletes)
 	}
 	switch {
+	case vp.IsSet(ToFQDNsActiveConnectionsTTL): // set by user
```

Comment:
I'm not sure whether `vp.IsSet()` is the best option here. Consider the case where the user *doesn't* configure `ToFQDNsActiveConnectionsTTL` but *does* configure `DNSGCJobInterval`. The 2x multiplier check here could be invalidated due to the _other_ configuration.

Maybe this whole switch statement could be simplified just to a check of `if vp.GetInt(ToFQDNsActiveConnectionsTTL) < int(2*defaults.DNSGCJobInterval) { ...}` with the override to ensure it is double the GC Job interval?

Reply:
I don't understand the scenario, how can the user specify `DNSGCJobInterval` ? I thought we only allow user to specify `ToFQDNsActiveConnectionsTTL`

Reply:
Oh, you're right, I don't know why I thought that was configurable. Usually most things are configurable even if they're hidden.

I guess the intent behind the switch statement is to specifically identify the case where a user has specified an invalid setting, and override + log in that case? That's reasonable.